inverse_test.go

```diff
@@ -70,6 +71,56 @@ func TestInverseTestAndAdd(t *testing.T) {
 	}
 }
 
+// Ensures an InverseBloomFilter can read and write successfully
+func TestInverseBloomFilter_ReadFrom(t *testing.T) {
```

Comment:
@Xopherus the new test. Good call, `capacity` and `size` were actually different.